### PR TITLE
97978 Ensure "Main Events Page" Shows up in Customer Homepage Settings

### PR DESCRIPTION
**Pull request by ggwicz**
**Ticket:** [**#97978**](http://central.tri.be/issues/97978)

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $name is never used | [Front_Page_View.php:20](https://github.com/moderntribe/the-events-calendar/blob/0353d7f8d811c86f673f31fcbdd3fa19a839313e/src/Tribe/Customizer/Front_Page_View.php#L20)
:warning: | The method parameter $r is never used | [Front_Page_View.php:45](https://github.com/moderntribe/the-events-calendar/blob/0353d7f8d811c86f673f31fcbdd3fa19a839313e/src/Tribe/Customizer/Front_Page_View.php#L45)
:warning: | The method parameter $pages is never used | [Front_Page_View.php:45](https://github.com/moderntribe/the-events-calendar/blob/0353d7f8d811c86f673f31fcbdd3fa19a839313e/src/Tribe/Customizer/Front_Page_View.php#L45)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Customizer/Front_Page_View.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ * @subpackage Customizer
+ * @since TBD
+ */
+class Tribe__Events__Customizer__Front_Page_View extends Tribe__Customizer__Section {
+
+	/**
+	 * PHP 5.2 method of creating "instances" of an abstract require this
+	 *
+	 * Note: This is the only required method for a Connector to work
+	 *
```
Comment (mitogh) [inline src/Tribe/Customizer/Front_Page_View.php]: missing `@since`

**Comment by mitogh** [comment]
Overall SGTM, however I think the commit message should describe the change instead to make it more easier so spot when debugging or by looking into the history.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $name is never used | [Front_Page_View.php:21](https://github.com/moderntribe/the-events-calendar/blob/8a8dbb27c7910e48eaf4e0a5485431e85d29a97c/src/Tribe/Customizer/Front_Page_View.php#L21)
:warning: | The method parameter $r is never used | [Front_Page_View.php:46](https://github.com/moderntribe/the-events-calendar/blob/8a8dbb27c7910e48eaf4e0a5485431e85d29a97c/src/Tribe/Customizer/Front_Page_View.php#L46)
:warning: | The method parameter $pages is never used | [Front_Page_View.php:46](https://github.com/moderntribe/the-events-calendar/blob/8a8dbb27c7910e48eaf4e0a5485431e85d29a97c/src/Tribe/Customizer/Front_Page_View.php#L46)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $name is never used | [Front_Page_View.php:21](https://github.com/moderntribe/the-events-calendar/blob/e9febc60172dea294b57b8d410d28e5ab07692a1/src/Tribe/Customizer/Front_Page_View.php#L21)
:warning: | The method parameter $r is never used | [Front_Page_View.php:46](https://github.com/moderntribe/the-events-calendar/blob/e9febc60172dea294b57b8d410d28e5ab07692a1/src/Tribe/Customizer/Front_Page_View.php#L46)
:warning: | The method parameter $pages is never used | [Front_Page_View.php:46](https://github.com/moderntribe/the-events-calendar/blob/e9febc60172dea294b57b8d410d28e5ab07692a1/src/Tribe/Customizer/Front_Page_View.php#L46)

> via jenkins/codesniffer-phpcs

**Comment by ggwicz** [comment]
Great points @mitogh, I've implemented your feedback and will keep the feedback about commit messages in mind going forward. This one is ready for another quick review! 🔍

**Pull request merged**